### ISSUE TITLE
Support for Arm64

### ISSUE THREAD

**Issue by mandolyte** [liamg/aminal#76]
**Describe the bug**
Using ChromeOS with Crostini which is a debian type linux. No binaries available for this architecture so I tried to build from source.

Bottom line after running `make install`, the last few lines output were: 
```
Architecture aarch64 is not supported by this installation script
Makefile:22: recipe for target 'install-tools' failed
make: *** [install-tools] Error 1
```
**To Reproduce**
1. Launch terminal on ChromeOS/Crostini
2. Run following for dependencies:
```
sudo apt-get install libgl1-mesa-dev
sudo apt-get install xorg-dev
```
3. Then run `go get -u github.com/liamg/aminal`
4. Then `cd $GOPATH/github.com/liamg/aminal`
5. Then `make install`
6. Output is:
```
go test -v ./...
?   	github.com/liamg/aminal	[no test files]
=== RUN   TestOffsets
--- PASS: TestOffsets (0.00s)
=== RUN   TestBufferCreation
--- PASS: TestBufferCreation (0.00s)
=== RUN   TestBufferWriteIncrementsCursorCorrectly
--- PASS: TestBufferWriteIncrementsCursorCorrectly (0.00s)
=== RUN   TestWritingNewLineAsFirstRuneOnWrappedLine
--- PASS: TestWritingNewLineAsFirstRuneOnWrappedLine (0.00s)
=== RUN   TestWritingNewLineAsSecondRuneOnWrappedLine
--- PASS: TestWritingNewLineAsSecondRuneOnWrappedLine (0.00s)
=== RUN   TestSetPosition
--- PASS: TestSetPosition (0.00s)
=== RUN   TestMovePosition
--- PASS: TestMovePosition (0.00s)
=== RUN   TestVisibleLines
--- PASS: TestVisibleLines (0.00s)
=== RUN   TestClearWithoutFullView
--- PASS: TestClearWithoutFullView (0.00s)
=== RUN   TestClearWithFullView
--- PASS: TestClearWithFullView (0.00s)
=== RUN   TestCarriageReturn
--- PASS: TestCarriageReturn (0.00s)
=== RUN   TestCarriageReturnOnFullLine
--- PASS: TestCarriageReturnOnFullLine (0.00s)
=== RUN   TestCarriageReturnOnFullLastLine
--- PASS: TestCarriageReturnOnFullLastLine (0.00s)
=== RUN   TestCarriageReturnOnWrappedLine
--- PASS: TestCarriageReturnOnWrappedLine (0.00s)
=== RUN   TestCarriageReturnOnLineThatDoesntExist
--- PASS: TestCarriageReturnOnLineThatDoesntExist (0.00s)
=== RUN   TestGetCell
--- PASS: TestGetCell (0.00s)
=== RUN   TestGetCellWithHistory
--- PASS: TestGetCellWithHistory (0.00s)
=== RUN   TestGetCellWithBadCursor
--- PASS: TestGetCellWithBadCursor (0.00s)
=== RUN   TestCursorAttr
--- PASS: TestCursorAttr (0.00s)
=== RUN   TestCursorPositionQuerying
--- PASS: TestCursorPositionQuerying (0.00s)
=== RUN   TestRawPositionQuerying
--- PASS: TestRawPositionQuerying (0.00s)
=== RUN   TestEraseLine
--- PASS: TestEraseLine (0.00s)
=== RUN   TestEraseLineToCursor
--- PASS: TestEraseLineToCursor (0.00s)
=== RUN   TestEraseLineAfterCursor
--- PASS: TestEraseLineAfterCursor (0.00s)
=== RUN   TestEraseDisplay
--- PASS: TestEraseDisplay (0.00s)
=== RUN   TestEraseDisplayToCursor
--- PASS: TestEraseDisplayToCursor (0.00s)
=== RUN   TestEraseDisplayFromCursor
--- PASS: TestEraseDisplayFromCursor (0.00s)
=== RUN   TestBackspace
--- PASS: TestBackspace (0.00s)
=== RUN   TestHorizontalResizeView
--- PASS: TestHorizontalResizeView (0.00s)
=== RUN   TestLine
--- PASS: TestLine (0.00s)
PASS
ok  	github.com/liamg/aminal/buffer	(cached)
=== RUN   TestColourTomlEncoding
--- PASS: TestColourTomlEncoding (0.00s)
=== RUN   TestColourTomlUnmarshalling
--- PASS: TestColourTomlUnmarshalling (0.00s)
=== RUN   TestKeyCombinations
--- PASS: TestKeyCombinations (0.00s)
PASS
ok  	github.com/liamg/aminal/config	(cached)
?   	github.com/liamg/aminal/glfont	[no test files]
?   	github.com/liamg/aminal/gui	[no test files]
?   	github.com/liamg/aminal/hints	[no test files]
=== RUN   TestParsing
--- PASS: TestParsing (0.00s)
PASS
ok  	github.com/liamg/aminal/sixel	(cached)
?   	github.com/liamg/aminal/terminal	[no test files]
?   	github.com/liamg/aminal/version	[no test files]
go vet -v
internal/race
internal/cpu
errors
math/bits
runtime/internal/sys
unicode/utf8
encoding
unicode
sync/atomic
image/color
internal/testlog
unicode/utf16
runtime/internal/atomic
container/list
crypto/internal/subtle
crypto/subtle
internal/bytealg
vendor/golang_org/x/crypto/cryptobyte/asn1
vendor/golang_org/x/net/dns/dnsmessage
internal/nettrace
math
strconv
crypto/rc4
runtime/cgo
runtime
sync
io
internal/singleflight
reflect
hash
bytes
math/rand
syscall
hash/crc32
sort
strings
bufio
encoding/binary
internal/syscall/unix
crypto/cipher
time
image
path
encoding/base64
regexp/syntax
github.com/liamg/aminal/vendor/github.com/gobuffalo/syncx
crypto/aes
internal/poll
image/internal/imageutil
image/draw
crypto
crypto/des
crypto/internal/randutil
crypto/sha512
crypto/hmac
crypto/md5
crypto/sha1
crypto/sha256
encoding/pem
vendor/golang_org/x/crypto/poly1305
regexp
vendor/golang_org/x/crypto/internal/chacha20
vendor/golang_org/x/crypto/curve25519
vendor/golang_org/x/text/transform
github.com/liamg/aminal/vendor/go.uber.org/zap/buffer
github.com/liamg/aminal/vendor/go.uber.org/atomic
os
vendor/golang_org/x/crypto/chacha20poly1305
github.com/liamg/aminal/vendor/go.uber.org/zap/internal/bufferpool
fmt
github.com/liamg/aminal/vendor/github.com/mcuadros/go-version
path/filepath
flag
context
encoding/hex
log
github.com/liamg/aminal/vendor/go.uber.org/zap/internal/exit
os/exec
encoding/json
compress/flate
io/ioutil
compress/gzip
github.com/liamg/aminal/vendor/github.com/kr/pty
github.com/liamg/aminal/vendor/github.com/joho/godotenv
github.com/liamg/aminal/vendor/github.com/BurntSushi/toml
github.com/liamg/aminal/vendor/github.com/go-gl/gl/all-core/gl
github.com/liamg/aminal/vendor/github.com/pkg/errors
github.com/liamg/aminal/vendor/github.com/go-gl/glfw/v3.2/glfw
github.com/liamg/aminal/vendor/github.com/gobuffalo/envy
net/url
math/big
vendor/golang_org/x/text/unicode/bidi
crypto/rand
encoding/asn1
net
crypto/rsa
crypto/elliptic
crypto/dsa
crypto/x509/pkix
crypto/ecdsa
vendor/golang_org/x/crypto/cryptobyte
vendor/golang_org/x/text/secure/bidirule
vendor/golang_org/x/text/unicode/norm
net/textproto
vendor/golang_org/x/net/http2/hpack
crypto/x509
vendor/golang_org/x/net/idna
mime
vendor/golang_org/x/net/http/httpguts
mime/quotedprintable
vendor/golang_org/x/net/http/httpproxy
net/http/internal
github.com/liamg/aminal/vendor/github.com/markbates/oncer
crypto/tls
mime/multipart
github.com/liamg/aminal/hints
net/http/httptrace
github.com/liamg/aminal/config
github.com/liamg/aminal/vendor/golang.org/x/image/math/fixed
github.com/liamg/aminal/sixel
github.com/liamg/aminal/vendor/go.uber.org/multierr
github.com/liamg/aminal/vendor/golang.org/x/image/font
github.com/liamg/aminal/vendor/github.com/golang/freetype/raster
github.com/liamg/aminal/vendor/go.uber.org/zap/internal/color
github.com/liamg/aminal/buffer
os/user
github.com/liamg/aminal/vendor/go.uber.org/zap/zapcore
net/http
github.com/liamg/aminal/vendor/github.com/golang/freetype/truetype
github.com/liamg/aminal/vendor/github.com/gobuffalo/packd
github.com/liamg/aminal/version
github.com/liamg/aminal/vendor/go.uber.org/zap
github.com/liamg/aminal/vendor/github.com/golang/freetype
github.com/liamg/aminal/vendor/github.com/gobuffalo/packr
github.com/liamg/aminal/glfont
github.com/liamg/aminal/vendor/github.com/riywo/loginshell
github.com/liamg/aminal/terminal
github.com/liamg/aminal/gui
github.com/liamg/aminal
# github.com/liamg/aminal
Checking file ./config.go
Checking file ./logger.go
Checking file ./main.go
which dep || curl -L https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  5110  100  5110    0     0  22892      0 --:--:-- --:--:-- --:--:-- 22914
Architecture aarch64 is not supported by this installation script
Makefile:22: recipe for target 'install-tools' failed
make: *** [install-tools] Error 1
```

**Expected behavior**
Clean install for aminal!

**Environment (please complete the following information):**
```
$ cat /etc/*release
PRETTY_NAME="Debian GNU/Linux 9 (stretch)"
NAME="Debian GNU/Linux"
VERSION_ID="9"
VERSION="9 (stretch)"
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
```

**Comment by liamg** [comment]
Thanks for the report. I'm travelling until Thursday so I'll take a proper look then.

In the meantime it looks like the issue is the dep install script. As a possible workaround you can look at installing/building [dep](https://github.com/golang/dep/) in a different way, or you can just try a `go build` if you have go set up.

**Comment by mandolyte** [comment]
Found the build target and it executed without problems:
```
$ go build -ldflags "-X github.com/liamg/aminal/version.Version=`git describe --tags`"
```

When executed, errors are:
```
$ cp aminal $GOBIN/bin
$ cd
$ aminal
1.5434457972810912e+09  info    aminal/main.go:28       Allocating pty...
1.5434457973070314e+09  info    aminal/main.go:56       Creating terminal...
GLFW: An uncaught error has occurred: PlatformError: Linux: Failed to watch for joystick connections in /dev/input: No such file or directory
GLFW: Please report this bug in the Go package immediately.
2018/11/28 22:56:37 PlatformError: Linux: Failed to open joystick device directory /dev/input: No such file or directory
1.543445797359083e+09   fatal   aminal/main.go:64       Render error: Failed to create window: Failed to create window: APIUnavailable: GLX: GLX extension not found
main.main
        /home/cecilnew/go/src/github.com/liamg/aminal/main.go:64
runtime.main
        /usr/local/go/src/runtime/proc.go:201
$
```

Thanks!

**Comment by liamg** [comment]
I think this may be fixed in #91 - please can you try again on master?
